src/api/controllers/PostController.ts

```diff
@@ -22,7 +22,7 @@ export class PostController {
     }
   }
 
-  @Get('id/:postId/')
+  @Get('id/:id/')
```

Comment:
why is this a necessary change?

Reply:
It's necessary bc in the validator that checks if the id is a UUID, the field is called `id`. Look at `api/validators/GenericRequests.ts`. So if this is called anything else, invoking `params.<that anything else>` will return none

Reply:
I'm p sure it's better to have one validator for a single UUID param field. The alternative is to make a validator class for `postId`, `userId`, and any other id that we use in the future, but maybe there's a way to work around this, idk